sales_team_security/models/res_partner.py

```diff
@@ -11,6 +13,64 @@ class ResPartner(models.Model):
 
     # add indexes for better performance on record rules
     user_id = fields.Many2one(index=True)
+    team_id = fields.Many2one(
```

Comment:
You introduce a new functionality here. Maybe this should be done in another module.